src/web/server/locales/en/validation.json

```diff
@@ -20,5 +20,5 @@
   "invalidPhoneNumber": "Enter a UK mobile number",
   "invalidEmail": "Enter an email address in the correct format, like name@example.com",
   "acceptTermsAndConditions": "Confirm that you’ve read and will comply with these terms and conditions",
-  "selectTextOrEmailChooseChannelForCode": "Select if you want to receive a code by text or email"
+  "selectTextOrEmailSendCode": "Select if you want to receive a code by text or email"
```

Comment:
Same here too.

Reply:
Gone with selectTextOrEmailChannel as we use channel for the field name.